www/src/data/sidebars/doc-links.yaml

```diff
@@ -442,6 +442,9 @@
                 - title: Adding Page Transitions with gatsby-plugin-transition-link
                   link: /docs/adding-page-transitions-with-plugin-transition-link/
                   breadcrumbTitle: gatsby-plugin-transition-link
+            - title: Adding a Redux Store
```

Comment:
I'm unsure if we want to make this super discoverable. Redux mostly only makes sense in a big client-side application. We don't want people to think you have to use Redux & React together with Gatsby, which was a common mistake made in "get started with react" tutorials.

I don't have a good idea where this belongs as we need it to clarify the "magic of gatsby".

Reply:
Yeh that makes sense. I've taken it out of the sidebar and am now linking to it at the end of the magic section (I meant to do that before!).